### PR TITLE
Groupon logo

### PR DESCRIPTION
**Pull request by jonzee**

### CODE REVIEW

File: src/sass/Groupon.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   img {
     width: 130px;
     height: auto;
-    float: right;
+    float: left;
```
Comment (michalmikolajczyk) [inline src/sass/Groupon.scss]: where is that style applied?

File: src/ts/Menu/index.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ export default function MenuVG(props) {
       <MenuItem className="menu-item" containerElement={<Link to="/page/guide" />} primaryText={i('Guides & Coaches')} />
       <MenuItem className="menu-item" containerElement={<Link to="/page/mechanic" />} primaryText={i('Mechanics')} />
       <MenuItem className="menu-item" containerElement={<Link to="/page/faq" />} primaryText={i('FAQ')} />
-      <MenuItem className="menu-item" containerElement={<Link to="/groupon" />} primaryText={<div style={{ height: 30 }}><img style={{ height: '100%', paddingTop: 7 }} src="https://assets1.grouponcdn.com/images/logos/powered_by_groupon.png" /></div>} />
+      <MenuItem className="menu-item" containerElement={<Link to="/groupon" />} primaryText={<div style={{ height: 20 }}><img style={{ height: '100%', paddingTop: 13 }} src="https://res.cloudinary.com/hkhuw4b7v/image/upload/v1500232121/groupon_logo_tpslhq.png" /></div>} />
```
Comment (michalmikolajczyk) [inline src/ts/Menu/index.tsx]: why that change in height?

File: src/ts/Page/Groupon.tsx [hunk 3]
```diff
@@ -13,17 +13,17 @@ export default function Grupon() {
         <div className="title">
           <div>VIKING</div>
           <div className="below">
-            GA<span className="replace">R<Raido /></span>AGE
+            GA<span className="replace"><Raido /></span>AGE
```
Comment (michalmikolajczyk) [inline src/ts/Page/Groupon.tsx]: why did you remove the character `R`? What do the bots see now?

Comment (jonzee) [inline src/ts/Page/Groupon.tsx]: there is svg fallback, bots see `garage`

File: src/ts/Menu/index.tsx (hunk 2 above)
Comment (jonzee) [inline src/ts/Menu/index.tsx]: because we changed logo

File: src/sass/Groupon.scss (hunk 1 above)
Comment (jonzee) [inline src/sass/Groupon.scss]: to groupon logo

**Comment by michalmikolajczyk** [comment]
- [ ] Move Groupon
